src/sentry/api/endpoints/external_team.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+from rest_framework import serializers, status
+from rest_framework.response import Response
+
+from sentry.api.bases.team import TeamEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import ExternalTeam, EXTERNAL_PROVIDERS
+
+logger = logging.getLogger(__name__)
+
+
+class ExternalTeamSerializer(serializers.ModelSerializer):
```

Comment:
@NisanthanNanthakumar I would suggest using the `CamelSnakeModelSerializer` base class here so we can keep Python using snake_case and the API interfaces using camelCase.